modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -46,6 +47,14 @@ function widget_admin_setup() {
 		wp_enqueue_script( 'widget-jetpack-search-filters' );
 	}
 
+	private function get_sort_types() {
+		return array(
+			'relevance|DESC' => esc_html__( 'Relevance', 'jetpack' ),
```

Comment:
Could you please use `esc_html_x` here to add context to the phrases? Something like:
```
esc_html_x( 'Relevance', 'Sort order specification - will follow the phrase "Sort by" in the search form controls', 'jetpack' )
```